examples/interactive_play.py

```diff
@@ -79,15 +79,15 @@ def append_text_to_image(image: np.ndarray, text: str):
     font_size = 0.5
     font_thickness = 1
     font = cv2.FONT_HERSHEY_SIMPLEX
-    blank_image = np.zeros(image.shape, dtype=np.uint8)
+    text_image = np.zeros(image.shape, dtype=np.uint8)
```

Comment:
```suggestion
    text_image = np.zeros_like(image)
```
You can specify the dtype if needed